source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -71,7 +71,7 @@ export abstract class Dialog {
             dialogFrameElement.onclick = (e) => {
                 // disable click bubbling from the frame element -- otherwise it'll get to the overlay, closing the
                 // dialog unexpectedly
-                if (this.preventLightDismissal) {
+                if (!this.preventLightDismissal) {
```

Comment:
Again, why this? It should be removed - not only is it not necessary but it breaks the dialog behavior when `preventLightDismissal` is false.

Reply:
@dclaux, without this prevention, dialog gets closed on image selection in Pic2card. We want the dialog to be available for further actions like file browse and adaptive card conversion.

Reply:
Indeed :) That is precisely why this should **always** be done, including when `preventLightDismissal` is false - that is basically what the comment in the code says.

Reply:
To be super clear - this is not about removing the `cancelBubble` statement, it is about removing the `if` statement.

Reply:
Nevermind - I had missed another comment where you explain precisely why you don't want cancelBubble. Please ignore this, I understand.